src/config/configManager.go

```diff
@@ -207,6 +207,15 @@ func LoadConfigFromFile() {
 		DBName:              viper.GetStringSlice("purge-old-db-entries.dbname"),
 	}
 
+	// recommend policy configurations
+	CurrentCfg.ConfigRecommendPolicy = types.ConfigRecommendPolicy{
+		Enable:                  viper.GetBool("recommend.enable"),
```

Comment:
There is no point of keeping the enable option... We can disable policy recommendation by setting OperationMode = NOOP